backend/tracim_backend/lib/core/content.py

```diff
@@ -847,6 +846,7 @@ def get_jpg_preview_path(
         :return: preview_path as string
         """
         file_path = self.get_one_revision_filepath(revision_id)
+        page = preview_manager_page_format(page)
         if page >= self.preview_manager.get_page_nb(file_path):
             raise Exception(
```

Comment:
I believe raise a generic Exception is a wrong idea

Reply:
You're right, that's an error (specific exception for this already exist in the same file)